source/_includes/header.liquid

```diff
@@ -0,0 +1,26 @@
+<header
+  class="sl-r-banner"
+  itemtype="http://schema.org/WPHeader"
+  itemscope="itemscope"
+  role="banner">
+  <div class="sl-c-alert sl-c-alert--special">
+    <div class="sl-l-container">
+      <h2 class="sl-c-alert-title">Black Lives Matter</h2>
+      {% renderTemplate 'md' %}
+        Sass stands with the protesters against police violence. We encourage our users to
+        **get in the streets and join them if you can**.
```

Comment:
Not urgent, but I think this sort of message might make sense as global data - maybe an `alert` data file that either has markdown content, or eg `none`, and then an `if` statement to show the message when the data is present? (but maybe that's getting ahead of ourselves on cleanup?)

Reply:
Fixed in https://github.com/oddbird/sass-site/commit/9b544fb71065fd3d693daded2332feb5e35f727c